content/en/serverless/guide/forwarder_extension_migration.md

```diff
@@ -0,0 +1,123 @@
+
+{{< tabs >}}
+{{% tab "Lambda Layer" %}}
+
+1. Instrument your [Python][1] or [Node.js][2] application by installing the Datadog Lambda Library.
+
+2. Add the Lambda Layer for the Datadog extension to your AWS Lambda function with the following ARN:
+
+    ```
+    arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:<EXTENSION_VERSION>
+    ```
+
+    Replace the placeholder values in the ARN as follows:
+    - Replace `<AWS_REGION>` with the same AWS region as your Lambda Function, for example, `us-east-1`.
+    - Replace `<EXTENSION_VERSION>` with the version of the Datadog Lambda Extension you would like to use, for example `7`. You can identify the current version by viewing the newest image tags in the [Amazon ECR repository][3].
+
+    **Note**: This Layer is separate from the Datadog Lambda Library. If you installed the Datadog Lambda Library as a Lambda Layer,
+    your function should now have two Lambda Layers attached.
+
+3. Add the environment variable `DD_API_KEY` and set the value to your Datadog API key on the [API Management page][4]. 
+
+4. Reference the [sample code][5] to submit a custom metric.
+
+[1]: /serverless/installation/python
+[2]: /serverless/installation/nodejs
+[3]: https://gallery.ecr.aws/datadog/lambda-extension
+[4]: https://app.datadoghq.com/account/settings#api
+[5]: /serverless/custom_metrics#custom-metrics-sample-code
+
+{{% /tab %}}
+{{% tab "Container Image" %}}
+
+. Install the Datadog Lambda Library by following the installation instructions for [Python][1] or [Node.js][2]. Use the installation instructions specifically for functions deployed as container images.
+
+2. Add the Datadog Lambda Extension to your container image by adding the following to your Dockerfile:
+
+```
+COPY --from=public.ecr.aws/datadog/lambda-extension:<TAG> /opt/extensions/ /opt/extensions
+```
+
+Replace `<TAG>` with either a specific version number (for example, `7`) or with `latest`. You can see a complete list of possible tags in the [Amazon ECR repository][3].
+
+3. Add the environment variable `DD_API_KEY` and set the value to your Datadog API key on the [API Management page][4]. 
+
+4. Reference the [sample code][5] to submit a custom metric.
+
+[1]: /serverless/installation/python
+[2]: /serverless/installation/nodejs
+[3]: https://gallery.ecr.aws/datadog/lambda-extension
+[4]: https://app.datadoghq.com/account/settings#api
+[5]: /serverless/custom_metrics#custom-metrics-sample-code
+
+{{% /tab %}}
+{{< /tabs >}}
+
+For more information, see the [Datadog Lambda Extension documentation][1].
+
+### Removing the Datadog Forwarder
+
+Remove subscriptions (TK)
+
+#### Prevent the Forwarder from attaching to new functions {#prevent-forwarder}
+
+When you deploy a new function, the Datadog Forwarder detects an unsubscribed function and automatically creates a new subscription. Depending on whether the Forwarder was configured automatically or manually, complete the following steps:
```

Comment:
I think the instructions here are the same regardless of whether the Forwarder was configured automatically or manually. The main difference with automatic is that the Forwarder will not automatically re-subscribe to your function's log groups if the Extension is installed on that function.

We should instead look to split this up based on the deployment method you are using (Serverless Framework, AWS SAM, AWS CDK, Custom)